src/pages/example.tsx

```diff
@@ -11,6 +21,79 @@ export default function Example() {
           Back to homepage
         </Button>
       </Link>
+
+      <Dropdown
+        options={[
+          {
+            value: 1,
+            label: 'Option one',
+          },
+          {
+            value: 2,
+            label: 'Option two',
+          },
+          {
+            value: 3,
+            label: 'Option three',
+          },
+        ]}
+      >
+        <Button icon={FaChevronDown} variant="secondary" iconPosition="right">
+          Menu Dropdown
+        </Button>
+      </Dropdown>
+
+      <Menu mode="horizontal">
+        <SubMenu
+          popupOffset={[-20, 0]}
```

Comment:
The offset doesn't seem correct. The menu is to far on the left. Check this again so that the arrow of the menu is at the center of the button.